app/forms/publish/accredited_provider_form.rb

```diff
@@ -34,7 +34,13 @@ def after_save
     def assign_attributes_to_model
       model.accrediting_provider_enrichments = [] if model.accrediting_provider_enrichments.nil?
 
```

Comment:
on line 14 I think we also validate by `words_count: { maximum: 100, message: :too_long }`

Reply:
Done: https://github.com/DFE-Digital/publish-teacher-training/pull/3573/commits/129554111fc3d7cdd765d12f38d2fb7b7bae7415